src/lucky_cli/browser_authentication_src_template.cr

```diff
@@ -1,3 +1,167 @@
+              ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/mixins/auth/password_resets/find_user.cr.ecr", io)
+            end
+            password_resets_dir.add_file("require_token.cr") do |io|
+              ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/mixins/auth/password_resets/require_token.cr.ecr", io)
+            end
+            password_resets_dir.add_file("token_from_session.cr") do |io|
+              ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/mixins/auth/password_resets/token_from_session.cr.ecr", io)
+            end
+          end
+        end
+      end
+      actions_dir.add_folder("password_reset_requests") do |password_reset_requests_dir|
+        password_reset_requests_dir.add_file("create.cr") do |io|
+          ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/password_reset_requests/create.cr.ecr", io)
+        end
+        password_reset_requests_dir.add_file("new.cr") do |io|
+          ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/password_reset_requests/new.cr.ecr", io)
+        end
+      end
+      actions_dir.add_folder("password_resets") do |password_resets_dir|
+        password_resets_dir.add_file("create.cr") do |io|
+          ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/password_resets/create.cr.ecr", io)
+        end
+        password_resets_dir.add_file("edit.cr") do |io|
+          ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/password_resets/edit.cr.ecr", io)
+        end
+        password_resets_dir.add_file("new.cr") do |io|
+          ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/password_resets/new.cr.ecr", io)
+        end
+      end
+      actions_dir.add_folder("sign_ins") do |sign_ins_dir|
+        sign_ins_dir.add_file("create.cr") do |io|
+          ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/sign_ins/create.cr.ecr", io)
+        end
+        sign_ins_dir.add_file("delete.cr") do |io|
+          ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/sign_ins/delete.cr.ecr", io)
+        end
+        sign_ins_dir.add_file("new.cr") do |io|
+          ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/sign_ins/new.cr.ecr", io)
+        end
+      end
+      actions_dir.add_folder("sign_ups") do |sign_ups_dir|
+        sign_ups_dir.add_file("create.cr") do |io|
+          ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/sign_ups/create.cr.ecr", io)
+        end
+        sign_ups_dir.add_file("new.cr") do |io|
+          ECR.embed("#{__DIR__}/../browser_authentication_app_skeleton/src/actions/sign_ups/new.cr.ecr", io)
+        end
+      end
+    end
+  end
+
+  private def emails_folder
+    LuckyTemplate.create_folder do |emails_dir|
```

Comment:
That's interesting that this already knows what directory its in. Pretty neat!

Reply:
It's not that it already knows, it's more that I'm making it clear that I expect it to be this way. You still need to add it to `Folder#insert_folder` and provide a name (which could be something other than "emails"). This is just so you can separate out the folders for better code organization or for reuse.